vital/logger/kwiver_logger_manager.cxx

```diff
@@ -101,34 +101,24 @@ ::kwiver_logger_manager()
   // we provide a method for creating these loggers.
 
   bool try_default(false);
-  std::string factory = std::getenv( PLUGIN_ENV_VAR );
```

Comment:
@aaron-bray This line was causing segfaults when the environment variable is not set.  The results was a large number of failing unit tests.  We need to check that the return value is not null before assigning the `char*` to a `std::string`.